R/filterReadsBam.R

```diff
@@ -0,0 +1,174 @@
+#' Write records from \code{infile} to \code{outfile} if they pass filter criteria.
+#'
+#' For each file in \code{infiles}, parse alignments, calculate read statistics
+#' and write the alignment to the corresponding output file from \code{outfiles}
+#' if the read passes all criteria defined by the filtering arguments.
+#' Filters are processed hierarchically: If a read does not pass a given
+#' filter, the remaining filters will not be examined and the processing
+#' continues with the next read.
+#' The filters are examined in this order: \code{minReadLength},
+#' \code{minAlignedLength}, \code{minAlignedFraction}, \code{minQscore},
+#' \code{maxFracLowConf}, \code{maxEntropy}.
+#'
+#' @param infiles Character vector with name(s) of the input bam file(s).
+#'     If \code{infiles} is a named vector, the names are used as sample names.
+#'     Otherwise, the each file will be names as \code{s1}, ..., \code{sN},
```

Comment:
... the files will be named ...
Actually, are the names necessary/useful for this function (since it's just file -> file, and these names will not be used to name or match to the outfiles)?

Reply:
that's true - I have removed the sentence from the documentation and do not test for unique names anymore in a07b2bdb